v2/kafka-to-pubsub/src/main/resources/kafka_to_pubsub_metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Kafka to Pub/Sub",
+  "description": "Streaming pipeline. Ingests JSON-encoded messages from Kafka to a Pub/Sub subscription",
```

Comment:
Ingests JSON-encoded messages from Kafka to a Pub/Sub topic

Reply:
fixed